package.json

```diff
@@ -13,18 +13,15 @@
     "url": "http://bitovi.com"
   },
   "scripts": {
-    "preversion": "npm test && npm run build",
-    "version": "git commit -am \"Update dist for release\" && git checkout -b release && git add -f dist/",
-    "postversion": "git push --tags && git checkout master && git branch -D release && git push",
+    "preversion": "npm test",
```

Comment:
Is this going to publish without a `dist` directory now?

Reply:
I don't think there's any need for a build step. Nothing to transpile, etc.